composer.json

```diff
@@ -12,7 +12,7 @@
         },
         {
             "name": "Thibault Duplessis",
-            "email": "thibault.duplessis@gmail.com"
+            "email": "<inactive>"
```

Comment:
remove the email field entirely from the object instead. This is making the composer.json file invalid.